### PR TITLE
New versions of nxlog

### PR DESCRIPTION
**Pull request by prensing**
Updated versions of nxlog. Previous version 2.10.2150 is no longer available online, so I removed it.

### CODE REVIEW

**Review by twangboy — CHANGES_REQUESTED** [review]
We need to leave the definition for unsupported/removed version. Just remove the installer portion of the definition (`installer` and `install_flags`). This makes the package remove only. It will allow users who have the old version installed to remove that version.

**Comment by prensing** [comment]
I put the old version back, removing the "installer" variables. However, the uninstall references the specific version of the MSI, which is no longer available for download. Is that going to work?

File: nxlog.sls [hunk 1]
```diff
@@ -1,10 +1,28 @@
 nxlog:
-  '2.10.2150':
+  '2.10.2100':
```
Comment (twangboy) [inline nxlog.sls:2]: 2150

**Review by twangboy — CHANGES_REQUESTED** [review]
For now we'll leave the uninstaller. In the future we need to use the product GUID for the uninstaller instead of an MSI at a URL. Then if the resource is removed, we can still uninstall.

File: nxlog.sls (hunk 1 above)
Comment (prensing) [reply nxlog.sls]: Sorry about that, I just checked in the change

**Review by twangboy — APPROVED** [review]

**Pull request merged**